red4ext-sys/cpp/glue/glue.hpp

```diff
@@ -132,11 +132,21 @@ const CBaseRTTIType* GetPropertyType(const CProperty* prop)
     return prop->type;
 }
 
+bool ExistsCName(const CName& cname)
+{
+    return strlen(CNamePool::Get(cname)) > 0;
+}
+
```

Comment:
I'd probably just implement this in Rust, since there's nothing here we can't easily do in Rust

Reply:
Refactored in d60c286